css/components/Button/button.css

```diff
@@ -0,0 +1,131 @@
+.btn {
+  border: none;
+  margin: 0.5rem;
+  padding: 1rem 2rem;
+  transition: background-color 0.3s;
+}
+
+.btn:hover {
+  background: var(--COLOR-SLATE-300);
+}
+
+/** Button Round **/
+
+.btn-rounded {
+  border-radius: 3rem;
+}
+
+/** Button Squared **/
+
+.btn-squared {
+  border-radius: 1rem;
+}
+
+/** Filled Buttons **/
+
+.btn-f-error {
```

Comment:
here as well - .btn-error-fill or .btn-error-solid . choose what sounds better to you.

Reply:
I was changing here too, 😅. To keep consistency, I have decided to use `btn-error-solid`. 

This would be good ?

Reply:
Yeah. 👍